modules/examples/eureka/setup.cfg

```diff
@@ -0,0 +1,29 @@
+[metadata]
```

Comment:
as per the latest standard, all the settings in setup.cfg are moved to `pyproject.toml`
ex: https://github.com/awslabs/autonomous-driving-data-framework/blob/main/modules/core/metadata-storage/pyproject.toml

Reply:
Removing this file.